### PR TITLE
Update index.json

### PR DESCRIPTION
**Pull request by stossy11**
I have read the CLA Document and I hereby sign the CLA

I added an app

### CODE REVIEW

**Comment by stossy11** [comment]
I have read the CLA Document and I hereby sign the CLA

File: store/index.json [hunk 1]
```diff
@@ -26,6 +26,23 @@
...
+      "url": "https://run3.io/popgame/fnaf/fnaf1/",
+      "gallery": [
+        "https://cdn.akamai.steamstatic.com/steam/apps/319510/ss_b5bf2127754d4e9bf6ab1c94e599d47f93a6708a.600x338.jpg",
+        "https://cdn.akamai.steamstatic.com/steam/apps/319510/ss_9d4e5e5aa2eb01b5c81cb5849d8049978c080742.600x338.jpg",
+        "https://cdn.akamai.steamstatic.com/steam/apps/319510/ss_cfe4d47366356fef6c7a8da2c0782b143584bbe4.116x65.jpg?t=1579635996"
+      ],
+      "desc": "Welcome to your new summer job at Freddy Fazbear's Pizza, where kids and parents alike come for entertainment and food as far as the eye can see! The main attraction is Freddy Fazbear, of course; and his two friends. They are animatronic robots, programmed to please the crowds! The robots' behavior has become somewhat unpredictable at night however, and it was much cheaper to hire you as a security guard than to find a repairman.",
+      "feat": "Ported by People"
+      "invert": false, 
+    }
```
Comment (Posandu) [inline store/index.json:45]: There's an error in this JSON

Comment (stossy11) [reply store/index.json:45]: can you help me fix it?

Comment (stossy11) [reply store/index.json:44]: hello?

**Comment by stossy11** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request closed without merging**